internal/pkg/cli/app_show.go

```diff
@@ -121,6 +129,45 @@ func (o *showAppOpts) description() (*describe.App, error) {
 	if err != nil {
 		return nil, fmt.Errorf("list jobs in application %s: %w", o.name, err)
 	}
+	//1st approach takes 2 seconds to display the table
+	for _, env := range envs {
+		deployedJobs, err := o.deployStoreClient.ListDeployedJobs(o.name, env.Name)
+		if err != nil {
+			return nil, fmt.Errorf("list deployed jobs %s: %w", o.name, err)
+		}
+		for _, job := range deployedJobs {
+			workloadEnvs[job] = append(workloadEnvs[job], env.Name)
+		}
+		deployedSvcs, err := o.deployStoreClient.ListDeployedServices(o.name, env.Name)
+		if err != nil {
+			return nil, fmt.Errorf("list deployed services %s: %w", o.name, err)
+		}
+		for _, svc := range deployedSvcs {
+			workloadEnvs[svc] = append(workloadEnvs[svc], env.Name)
+		}
```

Comment:
This seems like a great use case for [`errgroup.Group`](https://pkg.go.dev/golang.org/x/sync/errgroup) to get all of the jobs/services in parallel. I'd be curious to see how much it would slow down listing a single environment and how much it would speed up with multiple environments.

Reply:
Great suggestion, implemented and reduced response time by 66% 🤩